src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,8 +1,7 @@
 package core.mate.academy.model;
 
 /**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
+ * Add some custom fields that could be only in Bulldozer Do not remove no-args constructor
```

Comment:
The comment suggests adding custom fields specific to a Bulldozer. Consider adding fields such as `bladeWidth`, `enginePower`, or other Bulldozer-specific attributes.